lib/fastlane/helper/git_helper.rb

```diff
@@ -1,9 +1,12 @@
 module Fastlane
   module Actions
-    def self.git_log_between(pretty_format, from, to, include_merges)
+    GIT_MERGE_COMMIT_FILTERING_OPTIONS = [:include_merges, :exclude_merges, :only_include_merges].freeze
```

Comment:
That's fair - I definitely understand where you're coming from. 

My reasoning was that I couldn't think of a use case when I would want to ask `git` to find the set of commits that both _is_ and _isn't_ there. Even if it's valid to ask for, it's a strange and conflicted thing to request, so it felt better to me to keep that possible confusion out of all levels of the API.

If it proves to be a valuable thing to be able to ask for, we could always do as you say and make it possible later. What do you think?